src/basic/conf-files.c

```diff
@@ -369,7 +369,7 @@ int conf_files_list_dropins(
         assert(dirs);
 
         suffix = strjoina("/", dropin_dirname);
-        r = strv_extend_strv_concat(&dropin_dirs, (char**) dirs, suffix);
```

Comment:
Hmm, what we do here is precisely "concating"? All strings in the second strv are concated with one string and the first strv is extended to include the results.